CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+# 1.9.0 2021-06-28
+
+* Provides new ETL routine written in Java, that will replace all Jython scripts at some point [PR](https://github.com/qbicsoftware/etl-scripts/pull/85)
+* Support for nf-core pipeline result registration [PR](https://github.com/qbicsoftware/etl-scripts/pull/85)
+* Provides metadata validation for imaging data (OMERO etl). [PR](https://github.com/qbicsoftware/etl-scripts/pull/83)
```

Comment:
```suggestion
* Provides new ETL routine written in Java, that will replace all Jython scripts at some point [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/85)
* Support for nf-core pipeline result registration [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/85)
* Provides metadata validation for imaging data (OMERO etl). [(#85)](https://github.com/qbicsoftware/etl-scripts/pull/83)
```

I think it makes sense to stay consistent with other repositories concerning the style of the changelog. Apart from that this looks very nice :+1: